From: =?UTF-8?q?Rog=C3=A9rio=20Brito?= <rbrito@ime.usp.br>
Date: Sat, 12 May 2012 01:53:49 -0300
Subject: Fix include with the system's libav.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Thanks to Fabian Greffrath for the more elegant solution than the "brute
force" that I employed earlier.

Forwarded: yes
Last-Update: 2012-05-15
Reviewed-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
Signed-off-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
---
 libhb/decavcodec.c      |    4 +++-
 libhb/encavcodecaudio.c |    4 +++-
 2 files changed, 6 insertions(+), 2 deletions(-)

--- a/libhb/decavcodec.c
+++ b/libhb/decavcodec.c
@@ -38,7 +38,9 @@
 #include "hb.h"
 #include "hbffmpeg.h"
 #include "downmix.h"
-#include "libavcodec/audioconvert.h"
+
+struct AVAudioConvert;
+typedef struct AVAudioConvert AVAudioConvert;
 
 static void compute_frame_duration( hb_work_private_t *pv );
 static void flushDelayQueue( hb_work_private_t *pv );
--- a/libhb/encavcodecaudio.c
+++ b/libhb/encavcodecaudio.c
@@ -7,7 +7,9 @@
 #include "hb.h"
 #include "hbffmpeg.h"
 #include "downmix.h"
-#include "libavcodec/audioconvert.h"
+
+struct AVAudioConvert;
+typedef struct AVAudioConvert AVAudioConvert;
 
 struct hb_work_private_s
 {
